### PR TITLE
Push!

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Push

## Game Information
- **Title**: Push!
- **URL**: https://yyuta342.itch.io/push
- 
### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/push/port.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "push.zip",
+  "items": [
+    "Push.sh",
+    "push"
+  ],
+  "items_opt": null,
+  "attr": {
+    "title": "Megaball",
```
Comment (Cebion) [inline ports/push/port.json:10]: Is it push or megaball?

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/push/port.json (hunk 1 above)
Comment (tabreturn) [reply ports/push/port.json]: Woops! Fixed

File: ports/push/port.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "desc_md": null,
+    "inst": "Download the game free at https://yyuta342.itch.io/push, and place the push.pyxapp file in the push/gamedata folder.",
+    "inst_md": "Download the game free at [https://yyuta342.itch.io/push](https://yyuta342.itch.io/push), and place the *push.pyxapp* file in the `push/gamedata` folder.",
+    "genres": [
+      "puzzle"
+    ],
+    "image": {},
+    "rtr": false,
+    "exp": false,
+    "runtime": "pyxel_2.2.8_python_3.11",
```
Comment (kloptops) [inline ports/push/port.json:24]:
```suggestion
    "runtime": "pyxel_2.2.8_python_3.11.squashfs",
```

**Review by kloptops — CHANGES_REQUESTED** [review]
This one too.

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**